### PR TITLE
Add London as locality

### PR DESCRIPTION
**Pull request by owahltinez**
There are multiple Londons:
* `GB_ENG_E09000001`: City of London, which is a small region inside of what's colloquially known as London.
* `GB_UKI`: London Region, which is the City of London + Greater London and what most people think of when they hear London
* `N/A`: Greater London, which is the area surrounding London and excluding the City of London

London Region and Greater London are often confused. For example, the [Wikidata entry for London](https://www.wikidata.org/wiki/Q84) uses [Greater London for the Open Street Map relation ID](https://www.openstreetmap.org/relation/65606), which is incorrect.

As part of this change, we aggregate all the admin level 2 regions within the London Region to get epidemiological data for the new key `GB_ENG_LON`. We also equate the NUTS1 region `GB_UKI` to `GB_ENG_LON`, since they both refer to the same geographical region. This way, `GB_ENG_LON` will have a complete picture of epidemiological + static covariates (from Eurostat).

### CODE REVIEW

File: src/data/metadata.csv [hunk 1]
```diff
@@ -8887,15 +8888,15 @@ GB_SCT_S08000029,GB,United Kingdom,SCT,Scotland,S08000029,Fife,,,,
...
-GB_UKI,GB,United Kingdom,UKI,Greater London,,,,,E12000007,
-GB_UKJ,GB,United Kingdom,UKJ,South East,,,,,E12000008,
-GB_UKK,GB,United Kingdom,UKK,South West,,,,,E12000009,
+GB_UKC,GB,United Kingdom,UKC,North East England,,,,,E12000001,
+GB_UKD,GB,United Kingdom,UKD,North West England,,,,,E12000002,
+GB_UKE,GB,United Kingdom,UKE,"Yorkshire and the Humber, England",,,,,E12000003,
+GB_UKF,GB,United Kingdom,UKF,East Midlands England,,,,,E12000004,
+GB_UKG,GB,United Kingdom,UKG,West Midlands England,,,,,E12000005,
+GB_UKH,GB,United Kingdom,UKH,East of England England,,,,,E12000006,
+GB_UKI,GB,United Kingdom,UKI,London Region,,,,,(E12000007)|(Greater London),
```
Comment (themonk911) [inline src/data/metadata.csv:8897]: Not a request - but a question - is this a way of providing multiple match strings?

File: src/pipelines/epidemiology/config.yaml [hunk 2]
```diff
@@ -421,8 +417,6 @@ sources:
 
   - name: pipelines.epidemiology.gb_authority.Covid19UkL3DataSource
     test:
-      # TODO(owahltinez): skip test since it's very flakey
```
Comment (themonk911) [inline src/pipelines/epidemiology/config.yaml:424]: Unfortunately these tests remain flakey - I notice the API goes down after 5pm on Monday pretty regularly for instance, and sometimes the L3 data source simply times out (presumably if the answer is not in their cache yet).

File: src/data/metadata.csv [hunk 3]
```diff
@@ -8887,15 +8888,15 @@ GB_SCT_S08000029,GB,United Kingdom,SCT,Scotland,S08000029,Fife,,,,
...
-GB_UKH,GB,United Kingdom,UKH,East of England,,,,,E12000006,
-GB_UKI,GB,United Kingdom,UKI,Greater London,,,,,E12000007,
-GB_UKJ,GB,United Kingdom,UKJ,South East,,,,,E12000008,
-GB_UKK,GB,United Kingdom,UKK,South West,,,,,E12000009,
+GB_UKC,GB,United Kingdom,UKC,North East England,,,,,E12000001,
+GB_UKD,GB,United Kingdom,UKD,North West England,,,,,E12000002,
+GB_UKE,GB,United Kingdom,UKE,"Yorkshire and the Humber, England",,,,,E12000003,
+GB_UKF,GB,United Kingdom,UKF,East Midlands England,,,,,E12000004,
+GB_UKG,GB,United Kingdom,UKG,West Midlands England,,,,,E12000005,
+GB_UKH,GB,United Kingdom,UKH,East of England England,,,,,E12000006,
```
Comment (themonk911) [inline src/data/metadata.csv:8896]: Not sure "East of England England" works, perhaps there is no need for the additional England here (or maybe format similar to Yorkshire and the Humber.

**Review by themonk911 — CHANGES_REQUESTED** [review]
Thanks @owahltinez , I definitely fell into the category of people who assumed that Greater London contained City of London (i.e. Greater London == London Region). LGTM apart from minor comments.

File: src/data/metadata.csv (hunk 1 above)
Comment (owahltinez) [reply src/data/metadata.csv:8897]: Match string defaults to literal string match (against all *_name fields + string_match). If that fails, it falls back to regex match.

File: src/data/metadata.csv (hunk 3 above)
Comment (owahltinez) [reply src/data/metadata.csv:8896]: Good catch, I'll remove the superfluous name

File: src/pipelines/epidemiology/config.yaml (hunk 2 above)
Comment (owahltinez) [reply src/pipelines/epidemiology/config.yaml:424]: Got it, I'll keep the skip test then

**Comment by owahltinez** [comment]
All comments addressed

**Review by themonk911 — APPROVED** [review]

**Comment by owahltinez** [comment]
Thanks for reviewing!

**Pull request merged**